ocaml/src/asd_config.ml

```diff
@@ -50,3 +49,19 @@ let read_cfg cfg_file =
         "Interpreted the config as: %s"
         ([%show : Config.t] cfg)) >>= fun () ->
   Lwt.return config
+
+let retrieve_cfg_from_file cfg_file =
+  Lwt_extra2.read_file cfg_file >>= fun txt ->
+  Lwt_log.debug_f "Found the following config: %s" txt >>= fun () ->
+  retrieve_cfg_from_string txt
+
+
+
+let retrieve_cfg cfg_url =
+  let open Prelude.Url in
+  Lwt_io.eprintlf "config %s" (show cfg_url) >>= fun () ->
+  match cfg_url with
```

Comment:
I think we want a retrieve_from_url helper method iso doing this pattern match everywhere. (See also https://github.com/openvstorage/alba/pull/126/files#diff-15c6c580d31e8c48332322b747723af7R114 )